RELEASE_NOTES.md

```diff
@@ -1,8 +1,9 @@
 # Release notes
 
-## New in git-machete 3.1.2
+## New in git-machete 3.2.0
 
-- improved: slide-out can target branches with any number of downstream (child) branches
+- improved: `slide-out` can target branches with any number of downstream (child) branches
```

Comment:
I've added a minor bugfix in fork-point algorithm to `develop` in the meantime, pls rebase yourself :)